docs/data/charts/lines/lines.md

```diff
@@ -175,6 +175,20 @@ Different series could even have different interpolations.
 
 {{"demo": "InterpolationDemoNoSnap.js", "hideToolbar": true}}
 
+### Baseline
+
+The area chart draws a `baseline` on the Y axis `0`.
+This is useful as a base value, but customized visualizations may require a different baseline.
+
+To change it, simply provide the wanted `baseline` number of a series.
+It is also possible to provide `min` or `max` for the baseline to automatically follow the minimum or maximum value of the series.
```

Comment:
To encorage using min/max instead of dealing with putting a large value and using `axis.max`

```suggestion
To get the area filling the space above or below the line, set `baseline` to `"min"` or `"max"`.
It is also possible to provide a value to fix the baseline at the desired position.
```